### DISCUSSION TITLE
MsQuicConnection on Linux

### DISCUSSION THREAD

**Discussion by hros** [microsoft/msquic#1922] [Q&A] [answered]
`MsQuicConnection depends on `InterlockedIncrement` which is a Windows API
Is there support for Linux/POSIX?

**Comment by thhous-msft** [comment, marked as answer]
The C++ API in msquic.hpp is not part of the public API, and is not maintained. Its only there for test code, where we have shims for InterlockedIncrement on POSIX platforms. If trying to use MsQuic externally the C API is the only public API.

**Comment by thhous-msft** [comment]
I converted this to a discussion, as its not really an issue. Any discussion on anything in msquic.hpp can happen here, however at least for now its not a public API.
